python/src/uagents/mailbox.py

```diff
@@ -154,6 +170,38 @@ async def register_in_agentverse(
     return registration_response
 
 
+async def update_agent_details(
+    user_token: str,
+    agent_address: str,
+    agent_details: AgentUpdates,
+    agentverse: Optional[AgentverseConfig] = None,
+):
+    """
+    Updates agent details in Agentverse.
+
+    Args:
+        user_token (str): User token
+        agent_address (str): Agent address
+        agent_details (AgentUpdates): Agent details
+        agentverse (Optional[AgentverseConfig]): Agentverse configuration
+    """
+    agentverse = agentverse or AgentverseConfig()
+    try:
+        async with aiohttp.ClientSession() as session:
+            update_url = f"{agentverse.url}/v1/agents/{agent_address}"
+            async with session.put(
+                update_url,
+                data=agent_details.model_dump_json(),
+                headers={
+                    "Authorization": f"token {user_token}",
```

Comment:
is `"content-type": "application/json"` needed? (not sure about aiohttp, in requests it would be)